sdks/java/maven-archetypes/starter/pom.xml

```diff
@@ -79,5 +79,10 @@
       <artifactId>beam-sdks-java-core</artifactId>
       <scope>runtime</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
```

Comment:
The used but undeclared dependency is scoped to runtime.

2016-11-15T23:22:21.460 [WARNING] Used undeclared dependencies found:
2016-11-15T23:22:21.461 [WARNING]    org.slf4j:slf4j-api:jar:1.7.14:runtime